README.md

```diff
@@ -146,7 +146,7 @@ The default is `▉▊▋▌▍▎▏▎▍▌▋▊▉`. Set only one of these
 
 ### Search Controls
 
-The search tab performs a server-side search for text in metadata name fields.
+The search page performs a server-side search for text in IDv3 metadata fields.
```

Comment:
> Similar to search2, but organizes music according to ID3 tags.
https://subsonic.org/pages/api.jsp#search3

It doesn't say it _searches_ the ID3 tags; the documentation is ambiguous, as usual. It could be ID3v1, I suppose, although v2 has [been around since 1998](https://en.wikipedia.org/wiki/ID3#ID3v2).  Or, it could be _not_ searching ID3 tags, but just whatever they think "organizing according to tags" means.

I think what they mean, though, is that `search3` is supposed to search in ID3 tags -- which tags, they are -- again -- utterly ambiguous about. Suffice to say, as usual they don't really specify how servers are supposed to implement these endpoints, and it's pretty clear that every server is doing its own thing.

With regards to the README change: yes. I think it's a reasonably accurate reflection of the API documentation.

Note that the OpenSubsonic API documentation is _better_ than the Subsonic API documentation, but in this case does not provide any more clarity about defining expected server behavior. Also, I haven't found anything that says either Gonic or Navidrome implement [OpenSubsonic](https://opensubsonic.netlify.app/docs/endpoints) vs [Subsonic](https://subsonic.org/pages/api.jsp) API; they just say Subsonic-compatible. Whatever that means.